jekyll/_cci2/slack-orb-tutorial.adoc

```diff
@@ -78,16 +76,15 @@ jobs:
 workflows:
   send-notification:
     jobs:
-      - notify:
```

Comment:
I removed the context at this stage so that when it's added in later that triggers the pipeline, because the trigger pipeline button in app isn't available to GitLab projects